torch/onnx/symbolic_opset9.py

```diff
@@ -1037,44 +1037,34 @@ def _unique2(g, input, sorted, return_inverse, return_counts):
     globals()[name] = parse_args('v', 'i')(partial(sym_help._cast_func_template, v))
 
 
-@parse_args('v', 'i', 'v', 'v', 'b')
+@parse_args('v', 'i', 'v', 'v', 'v')
```

Comment:
Confused here, why change the pin_memory to value instead of bool?

And why remove the pin_memory check?

Reply:
In short, since pin_memory is not used, changing from bool to value matches how other unused parameters like layout and device are handled, and unblocks the above issue.

=============

In more details, this is related to the default argument issue we discussed regarding ScriptModule models. 

If pin_memory is not provided, it will be created as an empty prim::Constant in PTIR graph. 

For most of the ops, that part is not handled today. The onnx exporter has no idea what's the default argument value, so the default value has to be added explicitly in symbolic method. 

Some ops tried to resolve it by setting that param to 'v', and explicitly checking if it is empty.
https://github.com/pytorch/pytorch/blob/987f1ccf496a1f02b54378a6aeb50d9df0b3ac31/torch/onnx/symbolic_opset9.py#L826
https://github.com/pytorch/pytorch/blob/987f1ccf496a1f02b54378a6aeb50d9df0b3ac31/torch/onnx/symbolic_opset9.py#L831
Again, since pin_memory is not supported in onnx anyway, the latter checking part is ignored.